apps/admin_api/lib/admin_api/v1/controllers/blockchain_wallet_controller.ex

```diff
@@ -91,23 +84,11 @@ defmodule AdminAPI.V1.BlockchainWalletController do
     handle_error(conn, :invalid_parameter, "Invalid parameter provided. `address` is required.")
   end
 
-  defp add_balances_if_allowed(%Paginator{data: wallets, pagination: _} = paginator_wallets, conn) do
-    with {:ok, %{query: query}} <- authorize(:view_balance, conn.assigns, nil),
-         true <- !is_nil(query),
-         %Paginator{data: tokens} <- paginated_tokens(%{}),
-         {:ok, wallets_with_balances} <-
-           BlockchainBalanceLoader.wallet_balances(wallets, tokens) do
-      %{paginator_wallets | data: wallets_with_balances}
-    else
-      _ -> paginator_wallets
-    end
-  end
-
   defp respond_single(conn, blockchain_wallet) do
     render(conn, :blockchain_wallet, %{blockchain_wallet: blockchain_wallet})
   end
 
-  defp respond_multiple(conn, %Paginator{} = paged_wallets) do
+  defp respond_multiple(%Paginator{} = paged_wallets, conn) do
```

Comment:
Do you mind switching the params for respond_single above too? 🙏 Not very useful but better for consistency.